packages/manager/apps/pci-public-ip/src/pages/order/Order.page.tsx

```diff
@@ -51,6 +55,48 @@ export default function OrderPage(): JSX.Element {
     setSteps(initStartupSteps());
   }, []);
 
+  useEffect(() => {
```

Comment:
As a global improvement, I'd put the logic into dedicated functions that can enhance the `useEffects` readability.
By the way, I feel that you don't need effects if logic only depends from static params or state.